index.bs

```diff
@@ -412,6 +418,8 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
   :: 1. Return |platformObject|'s [=relevant global object=]'s [=associated Document=]'s [=Document/post-prerendering activation steps list=].
 </dl>
 
+Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which is initially a {{DOMHighResTimeStamp}} with a time value equal to zero. This is used to keep the time of immediately after [=Prompt to unload|prompting to unload=] the [=active document=] of the previous [=browsing context=].
```

Comment:
s/equal to/of/ ?

Reply:
Also nit: shorter lines would be easier to review

Reply:
Should the "This is used" part be in a note?

Reply:
Nit: s/time of/time/

Reply:
Done.